src/infcholesky.jl

```diff
@@ -18,6 +18,8 @@ end
 AdaptiveCholeskyFactors(A::AbstractMatrix{T}) where T = AdaptiveCholeskyFactors(MemoryLayout(A), A)
 MemoryLayout(::Type{AdaptiveCholeskyFactors{T,DM,M}}) where {T,DM,M} = AdaptiveLayout{typeof(MemoryLayout(DM))}()
 
+copy(A::AdaptiveCholeskyFactors) = AdaptiveCholeskyFactors(copy(A.data), copy(A.ncols))
+copy(A::AdjOrTrans{T,<:AdaptiveCholeskyFactors}) where T = copy(parent(A))'
```

Comment:
It should make a `Transpose` if `A isa Transpose`

Reply:
done